inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/fieldtype/strategy/ClickHouseFieldTypeStrategy.java

```diff
@@ -31,12 +34,23 @@ public class ClickHouseFieldTypeStrategy implements FieldTypeMappingStrategy {
 
     private final FieldTypeMappingReader reader;
 
+    private final static String NULLABLE_PATTERN = "^NULLABLE\\((.*)\\)$";
+
+    private static final Pattern PATTERN = Pattern.compile(NULLABLE_PATTERN);
+
     public ClickHouseFieldTypeStrategy() {
         this.reader = new FieldTypeMappingReader(DataNodeType.CLICKHOUSE);
     }
 
     @Override
     public String getFieldTypeMapping(String sourceType) {
+        // support clickHouse field type special modifier Nullable
+        if (StringUtils.isNotBlank(sourceType)) {
+            Matcher matcher = PATTERN.matcher(sourceType.toUpperCase());
+            if (matcher.matches()) {
+                sourceType = matcher.group(1);
```

Comment:
After retaining the first group, the latter are discarded. Will there be any other problems?

Reply:
Okay, I found the example in UT. Please add an example comment before `sourceType = matcher.group(1);`, thanks.